src/meta/buying_options.yaml

```
@@ -1,18 +1,82 @@
-carrot:
+golden_carrot:
   type: item
   name: Golden Carrots
   item: minecraft:golden_carrot
   price: 200
   amount: 8
+torch:
+  type: item
+  name: Torches
+  item: minecraft:torch
+  price: 100
+  amount: 32
+hopper:
+  type: item
+  name: Hoppers
+  item: minecraft:hopper
+  price: 500
+  amount: 16
```

```suggestion
  name: Hoppers
  item: minecraft:hopper
  price: 500
  amount: 4
```
500 for 16 seems ridiculously cheap. Maybe something like this will be more balanced? Remember, people have like 100k+ gurkcoins from quick mining sessions.

---

I am okay with cutting it in half and saying 8. You can't do anything with 4 hoppers. What do you think?

---

An iron ingot is worth 50 in the shredder, so a single hopper would be worth 150. 150*8=1200. Then we add 64 (8*8) wood, and we're looking at a *lot* of value, something like 1500 coins. There's also a pretty large convenience factor.

I think 8 hoppers should cost 1.5k or 2k. Either that, or we do 4 hoppers for 750 or 1k.